spec.html

```diff
@@ -27849,8 +27849,8 @@ <h1>Date ( _value_ )</h1>
             1. Else,
               1. Let _v_ be ? ToPrimitive(_value_).
               1. If Type(_v_) is String, then
-                1. Let _tv_ be the result of parsing _v_ as a date, in exactly the same manner as for the `parse` method (<emu-xref href="#sec-date.parse"></emu-xref>). If the parse resulted in an abrupt completion, _tv_ is the Completion Record.
-                1. ReturnIfAbrupt(_tv_).
+                1. Assert: parsing of String value _v_ does not return an abrupt completion.
```

Comment:
I actually liked @anba's wording better here. I think it's  useful to underscore that it's _because_ _v_ is a string that Date.parse won't throw (current text could imply that Date.parse doesn't throw ever or the string is constructed in a particular way so as not to throw).

Reply:
Makes sense.